### ISSUE TITLE
[BUG] Filenames with foreign letters like "äöü" will cause corrupted saves and crashes. - Windows

### ISSUE THREAD

**Issue by neoacix23** [opentoonz/opentoonz#666]
Filenames with foreign letters like "äöü" will cause corrupted saves and crashes.

**Comment by manongjohn** [comment]
I've discovered that it isn't really the filename that is the issue, it's the level name.  In the .tnz file, if foreign characters are in a level name and that level name is not encapsulated in "" (double-quotes), there are problems loading the file.

I noticed in a user's .tnz file I fixed, sometimes the level name with foreign characters was double-quoted and sometimes not.  The problem is when it is not.

One way to fix this would be to ensure when writing out the level name to the .tnz file, it should always be double-quoted regardless if there are foreign characters in it.  Unfortunately, this fix would only work for new scenes going forward.  

Existing scenes with this issue would either need to be manually corrected by the owner or we'd need to fix how the .tnz file is being read in and loaded.
